include/Beman/Optional26/optional.hpp

```diff
@@ -165,10 +165,11 @@ namespace std {
 */
 
 #include <compare>
-#include <utility>
+#include <format>
```

Comment:
does this need to be in a macro guard for modes below c++20 or compilers that don't support?

Reply:
Then we can check against `__cpp_lib_format_ranges`, because we don't only want `<format>`, but also changes proposed in P2585R1.

Btw, currently the project is configured to compile with C++23 :D We can switch to C++20 in https://github.com/beman-project/Optional26/issues/16. From a quick test, the `main` implementation won't compile with C++17.